tests/Auth/bot-authentication/Akhenaten.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+
+namespace AuthenticationBot
+{
+    public class Akhenaten : ComponentDialog
```

Comment:
👽

Reply:
@EricDahlvang I needed an obscure name in the test code to be sure I was definitely running the code - he was a pharaoh - very influential they say